videoroom/6_ImplementingClientsApplication.md

```diff
@@ -372,10 +372,10 @@ feel free to refer to the implementation of the video room's client-side availab
 [here](https://github.com/membraneframework/membrane_videoroom_tutorial/tree/template/end/assets/src).
 
 Now, finally, you should be able to check the fruits of your labor!
-Please run:
+Please run (EXTERNAL_IP is needed only if you develop inside docker):
```

Comment:
I believe EXTERNAL_IP is also needed in case you wanted the videoroom to be accessible from your local network